cmd/bbolt/command_surgery_cobra.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.etcd.io/bbolt/internal/common"
+	"go.etcd.io/bbolt/internal/surgeon"
+)
+
+var (
+	surgeryTargetDBFilePath string
+	surgeryPageId           uint64
+	surgeryStartElementIdx  int
+	surgeryEndElementIdx    int
+)
+
+func newSurgeryCobraCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "surgery <subcommand>",
+		Short: "surgery related commands",
+	}
+
+	cmd.AddCommand(newSurgeryClearPageElementsCommand())
+
+	return cmd
+}
+
+func newSurgeryClearPageElementsCommand() *cobra.Command {
+	clearElementCmd := &cobra.Command{
+		Use:   "clear-page-elements [options]",
+		Short: "Clears elements from the given page",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("db file path not provided")
+			}
+			if len(args) > 1 {
+				return errors.New("too many arguments")
+			}
+			return nil
+		},
+
+		RunE: surgeryClearPageElementFunc,
+	}
+
+	clearElementCmd.Flags().StringVar(&surgeryTargetDBFilePath, "output", "", "path to the target db file")
+	clearElementCmd.Flags().Uint64VarP(&surgeryPageId, "pageId", "", 0, "page id")
+	clearElementCmd.Flags().IntVarP(&surgeryStartElementIdx, "from", "", 0, "start element index (included) to clear, starting from 0")
+	clearElementCmd.Flags().IntVarP(&surgeryEndElementIdx, "to", "", 0, "end element index (excluded) to clear, starting from 0, -1 means to the end of page")
+
+	return clearElementCmd
+}
+
+func surgeryClearPageElementFunc(cmd *cobra.Command, args []string) error {
+	srcDBPath := args[0]
+
+	if err := copyFile(srcDBPath, surgeryTargetDBFilePath); err != nil {
```

Comment:
1. Do we have validation `surgeryTargetDBFilePath` is not empty 
(or logic to defaulting it to [srcDBPath.out] if it does not exists] ?

2. Do we have protection to not override accidentally existing file (I think we should be on the safe side with this tool here).

Reply:
The `--output ` must not exist when executing this command, otherwise fail the command. The validation is included `cmd/bbolt/utils.go`